.travis.yml

```diff
@@ -4,37 +4,25 @@
 # (as opposed to this top-level package which contains the tests),
 # see the "travis.yml.TEMPLATE" file.
 
-language: python
-sudo: false
+language: python  # Not really, because we're running Docker, but we need something
+services:
+  - docker
+sudo: true
 python:
-  - 2.7
+  - 3.6
 os:
   - linux
 matrix:
   fast_finish: true
 notifications:
   email: false
+  slack:
+    secure: G7gehYccV90HLQqCQKUCryeyM/S2fkAraZ+Kq2VUSMXfu+ZQ0nI6a8dtKh8Q1a9UOenTaFpx9sm5WMi2omNxsvWrtlhUCVn7S55k3rbyp0mhmncxM+N6738LHWBR0ach2uRa0A4qtDBQznjkButJhsWX/S2pxsi2xiu6ryhZaJY8CfF07VMUf5rvdWzFsvlSNOrG/uYzCwPtqBy9lb+RLwCGE/XbxuYI4D+uTkhr1dY7HFMpd2Dll8083LJQH7ewEkdt3Zr6yD/8ztztVsTFluGZ1M6tLhQqDLaBI8fVEYDJa6O3igGN8gvZaqAg+C9MasPbwFfp7h5Z2xHJytVcgTaWCIRpDMUHjBuunj8rDP+c45+a/yVpLmKKmbe06KfJh5NNzEA0wSuD/XPnrJ4GZ7UdlvfBghxdxTfhB0+eLT4oWzUDsc2ekX+WvAr+Igp+kZEfqgriVzqm07Ano2gcqMIgq37Ee+z35L72NiFSXHicaJLA5SDZ6fpQh9jK777vWXRZin6U+XBQBzVEnOacx6Wv6KK2rof1YX+Unwm3DW7DWvMVT7SzP6vWtswWojxmQvDKK9ZeO+MzSf/UzvNGHeuVHnwQyTtsguD/TcPWYlckksvApOBE44SGQ1ex5bGCfYk/20GPpf1fusoC+ODf7yVHHvp5CDIDZZ7dYLCinFM=
```

Comment:
Where is this slack notification pointing to? That is to #pipe2d slack channel? Perhaps a short comment above confirming that would be useful.